### PR TITLE
Feature/application form UI

### PR DESCRIPTION
**Pull request by isurupathumherath**
Updated error validation in Contact form & Created Application Form

### CODE REVIEW

**Comment by rusiruavb** [comment]
@isurupathumherath CI/ CD is failed. Run the build in your local machine and check why it is failing.

File: src/interfaces/JoinUsFormInterface.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  selfIntroduction?: string;
+  reason?: string;
+  linkedInProfile?: string;
+  githubProfile?: string;
+  blog?: string;
+  experiences?: string;
+  challengesAndProblems?: string;
+  timeline?: string;
+  skills?: string[];
+  formErrors: {
```
Comment (rusiruavb) [inline src/interfaces/JoinUsFormInterface.ts:18]: No need to have `formErrors` object. Now we use **Yup** package to handle validations. Remove the `formErrors` object and test the feature again.

File: src/pages/contact/contact-form-section.tsx [hunk 2]
```diff
@@ -1,177 +1,112 @@
 import React from 'react';
 import { IProps, IState } from '../../interfaces/ContactFormInterface';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as Yup from 'yup';
 
 const emailRegEX = RegExp(
   /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/
```
Comment (rusiruavb) [inline src/pages/contact/contact-form-section.tsx:8]: Remove `emailRegEX` if it is not use.

File: src/pages/join_us/join-us-form-section.tsx [hunk 3]
```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import { IProps, IState } from '../../interfaces/JoinUsFormInterface';
+import Select from 'react-select';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as Yup from 'yup';
+
+const skillOptions = [
```
Comment (rusiruavb) [inline src/pages/join_us/join-us-form-section.tsx:8]: Remove `skillOptions`. Use React select. I will look in to the react select issue

File: src/pages/join_us/join-us-form-section.tsx [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+  { value: 'Public Speaking', label: 'Public Speaking' },
+  { value: 'Designing', label: 'Designing' },
+  { value: 'Content Writing', label: 'Content Writing' },
+  { value: 'Video Editing', label: 'Video Editing' },
+  { value: 'Web Development', label: 'Web Development' },
+  { value: 'UI/UX', label: 'UI/UX' },
+  { value: 'Community building', label: 'Community building' }
+]
+
+const emailRegEX = RegExp(
```
Comment (rusiruavb) [inline src/pages/join_us/join-us-form-section.tsx:28]: `emailRegEX` is not used. Remove it.

File: src/pages/join_us/hero-section.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      <div className="col-sm-12 col-md-7 col-lg-7">
+        <h1 className="contact-us-header-title">Join with Us</h1>
+        <p className="contact-us-header-description">
+          Fill your application form today
+        </p>
+      </div>
+
+      <div className="col-sm-12 col-md-5 col-lg-5">
+        <div className="d-felx justify-content-center">
+          <img src="/assets/join_us_header.svg" alt="home header" className="contact-us-header-img" />
```
Comment (rusiruavb) [inline src/pages/join_us/hero-section.tsx:16]: We need to change the header image for the application.

**Review by rusiruavb — CHANGES_REQUESTED** [review]

**Review by rusiruavb — APPROVED** [review]

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**